cmd/cmd.go

```diff
@@ -939,6 +1036,35 @@ func RunServer(cmd *cobra.Command, _ []string) error {
 	return server.Serve(ln, origins)
 }
 
+func getBase64Image(filePath string) (string, error) {
+	_, err := os.Stat(filePath)
```

Comment:
This is unnecessary. Calling `os.Open` is better and not subject to TOCTOU problems